### PR TITLE
PW-431

### PR DESCRIPTION
**Pull request by IvanProdaiko**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+update-services:
+	test "$(KUBE_NAMESPACE)"
+	$(KUBECTL_CMD) apply -f devops/k8s/service.yml
+
+update-sa: ## push service account (KSA) to kube
+	$(KUBECTL_CMD) apply --record -f devops/k8s/sa.yml
+	$(KUBECTL_CMD) apply --record -f devops/k8s/rbac.yml
+
+update-psp: ## push pod security policy to kube
+	$(KUBECTL_CMD) apply --record -f devops/k8s/psp.yml
```
Comment (alexdicianu) [inline Makefile:61]: Let's get rid of the pod security policy. It's already deprecated in kube and going to be removed as a feature soon.

File: devops/k8s/configmaps/production/config/job-manager.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+job-creation-pubsub-project: pantheon-internal
+job-creation-pubsub-topic: job-manager-trigger
+job-creation-pubsub-attempt-topic-creation: false
+
+database:
+  project_id: pantheon-internal
+  spanner_instance_id: job-manager
+  env: production
+  migrations-dir: database/migrations
+  seed-data-dir: database/seed-data
```
Comment (alexdicianu) [inline devops/k8s/configmaps/production/config/job-manager.yml:30]: Do we need these for spanner databases?

File: devops/k8s/templates/sandbox-kubeconfig [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    namespace: tekton-pipelines
+    user: tekton-01-runner
+  name: gke_pantheon-sandbox_us-central1_sandbox-tekton-01
+current-context: gke_pantheon-sandbox_us-central1_sandbox-tekton-01
+kind: Config
+preferences: {}
+users:
+- name: tekton-01-runner
+  user:
+   token: __TOKEN__
```
Comment (alexdicianu) [inline devops/k8s/templates/sandbox-kubeconfig:19]: This can definitely go :)

**Review by alexdicianu — COMMENTED** [review]
Looking good for the most part. We should get rid of all the MySQL / Cloud SQL implementations and shift to instantiating a spanner client.

File: devops/k8s/configmaps/production/config/job-manager.yml (hunk 2 above)
Comment (IvanProdaiko) [reply devops/k8s/configmaps/production/config/job-manager.yml:30]: @alexdicianu so there is a migrations in spanner https://github.com/golang-migrate/migrate/blob/master/database/spanner/spanner.go

File: pkg/db/database.go [hunk 4]
```diff
@@ -0,0 +1,82 @@
+package db
+
+import (
+	"fmt"
+
+	_ "github.com/go-sql-driver/mysql"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	_ "github.com/googleapis/go-sql-spanner"
+	"github.com/jmoiron/sqlx"
```
Comment (alexdicianu) [inline pkg/db/database.go:9]: Not sure about all of these libraries. I would start with the google recommended libs, the ones they maintain and then look elsewhere only if those are lacking features we need or if we find serious bugs. They have a ton of examples below:

* https://github.com/GoogleCloudPlatform/golang-samples/blob/2e53f76946de2ae3913e8f7440866deb335701dc/spanner/spanner_snippets/spanner/spanner_query_data.go
* https://github.com/GoogleCloudPlatform/golang-samples/tree/2e53f76946de2ae3913e8f7440866deb335701dc/spanner/spanner_snippets/spanner

File: .circleci/config.yml [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+      - run: make setup-ar build-docker push
+
+  # Deploys the latest build to Kube
+  deploy:
+    executor: deploy-toolbox
+    steps:
+      - restore-workspace
+      - load-build-num
+      - run: UPDATE_GCLOUD=false make deps-circle-kube
+      - run: make deploy check-deployment-status KUBE_NAMESPACE=production CLUSTER_DEFAULT=general-01
```
Comment (alexdicianu) [inline .circleci/config.yml:102]: Let's comment this line for now until we have the production resources ready.

**Pull request merged**